packages/common/test-configs/jest-config-base.ts

```diff
@@ -5,7 +5,7 @@ const config: InitialOptionsTsJest = {
   preset: 'ts-jest/presets/js-with-ts',
 
   testMatch: ['**/*.test.(js|jsx|ts|tsx)'],
-  transformIgnorePatterns: ['/node_modules/(?!(@patternfly)/)'],
+  transformIgnorePatterns: ['/node_modules/'],
```

Comment:
FYI this was one of the reasons of the slowdowns, since we instructed `ts-jest` to transform all PatternFly packages by default.